six/modules/c++/six/source/NITFReadControl.cpp

```diff
@@ -418,6 +418,21 @@ void NITFReadControl::load(nitf::IOInterface& ioInterface,
             }
         }
 
+        // SIDD 1.1 needs to read LUT directly from NITF
+        if (currentInfo->getData()->getDataType() == DataType::DERIVED &&
+            currentInfo->getData()->getVersion() == "1.1.0")
+        {
+            nitf::LookupTable nitfLut =
+                    subheader.getBandInfo(0).getLookupTable();
+
+            //getDisplayLUT will not return NULL for 1.1
+            currentInfo->getData()->getDisplayLUT().reset(
```

Comment:
So rather than relying/hoping that someone always does this and remembers to call `interleaveTable()`, what about just adding a `six::LUT` constructor that takes in a `nitf::LookupTable` directly and does the right thing for you?